README.md

```diff
@@ -66,7 +66,7 @@ browser-based IDE. Here are some example contracts:
 
 ## Documentation
 
-The Solidity documentation is hosted at [Read the docs](https://docs.soliditylang.org).
+The Solidity documentation is hosted [here](https://docs.soliditylang.org) using **Read the Docs**.
```

Comment:
Link titles like `here` or `link` are bad for screen readers. The link should be a description of what it's linking to.

Reply:
Alright, I made some changes